<launch>
	<param name="use_sim_time" value="true"/>
	
	<!-- <node pkg="rosbag" type="play" name="player" args="&#45;&#45;clock /home/amsl/ozaki/bagfiles/tsukuba/2018&#45;09&#45;15&#45;16&#45;11&#45;46.bag &#45;s 250"/> -->
	<node pkg="rosbag" type="play" name="player" args="--clock /home/amsl/ozaki/bagfiles/tsukuba/2018-10-05-16-27-47.bag"/>	<!-- with zed_points -->

	<node pkg="rviz" type="rviz" name="rviz" args="-d /home/amsl/.rviz/tsukuba2018.rviz"/>
	
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>
	
	<node pkg="tsukuba2018" type="tf_broadcaster" name="tf_broadcaster"/>

	<node pkg="tsukuba2018" type="pc_transformer" name="velodyne2localmap"/>

	<node pkg="tsukuba2018" type="occupancygrid_lidar" name="occupancygrid_lidar" output="screen"/>
	<node pkg="tsukuba2018" type="occupancygrid_store" name="occupancygrid_store" output="screen"/>
	
	<!-- <node pkg="grass2grid" type="grass2grid" name="grass2grid"/> -->
	
	<node pkg="tsukuba2018" type="pc_transformer" name="zed_grass2localmap">
		<remap from="/velodyne_points" to="/zed_grasspoints"/>
		<remap from="/velodyne_points/transformed" to="/zed_grasspoints/transformed"/>
	</node>
	<node pkg="tsukuba2018" type="pc_transformer" name="zed_road2localmap">
		<remap from="/velodyne_points" to="/zed_roadpoints"/>
		<remap from="/velodyne_points/transformed" to="/zed_roadpoints/transformed"/>
	</node>
	
	<node pkg="tsukuba2018" type="occupancygrid_zed" name="occupancygrid_zed" output="screen"/>
	
	<node pkg="tsukuba2018" type="longest_path_detection" name="longest_path_detection" output="screen"/>

</launch>
